lib/ui/settings/SettingsHomePanel.dart

```diff
@@ -634,25 +694,32 @@ class _SettingsHomePanelState extends State<SettingsHomePanel> implements Notifi
       ));
     }
     else {
-      contentList.add(ToggleRibbonButton(
-          height: null,
-          borderRadius: _borderRadiusFromIndex(0, 2),
-          label: Localization().getStringEx("panel.settings.home.covid19.exposure_notifications", "Exposure Notifications"),
-          toggled: (_healthUser?.exposureNotification == true),
-          context: context,
-          onTap: _onExposureNotifications));
-
-      contentList.add(ToggleRibbonButton(
-          height: null,
-          borderRadius: _borderRadiusFromIndex(1, 2),
-          label: Localization().getStringEx("panel.settings.home.covid19.provider_test_result", "Health Provider Test Results"),
-          toggled: (_healthUser?.consent == true),
-          context: context,
-          onTap: _onProviderTestResult));
-
-      contentList.add(
-        Padding(padding: EdgeInsets.only(left: 8, top: 16), child:
-          _buildCovid19KeysSection(),));
+      List<dynamic> codes = FlexUI()['settings.covid19'] ?? [];
+      for (int index = 0; index < codes.length; index++) {
+        String code = codes[index];
+        BorderRadius borderRadius = _borderRadiusFromIndex(index, codes.length);
+        if (code == 'exposure_notifications') {
+          contentList.add(ToggleRibbonButton(
+              height: null,
+              borderRadius: borderRadius,
```

Comment:
@TodorBachvarov Please review this change is correct. It's related to a previous fix made by you. Awaiting for confirmation